### ISSUE TITLE
D'sparil attacks aren't loud

### ISSUE THREAD

**Issue by ioan-chera** [team-eternity/eternity#393]
D'sparil attacks must make their sounds at full volume, which they fail in Eternity

**Comment by Altazimuth** [comment]
MY EARS.

**Comment by sinisterseed** [comment]
Wait a sec, "full volume" as in **_max_** volume, ignoring the user's settings?

If that's the case, I think it's better to avoid doing this and keep the volume level the same. This reminds me of overlapping sound effects in PrBoom... very painful...

**Comment by ioan-chera** [comment]
No. I mean full volume, just like the sight and active sounds of D'sparil.

**Comment by sinisterseed** [comment]
Oh, alright then, I got a small heart attack initially thinking that the idea was to make him blast the user's ears off lol.
